src/Illuminate/Foundation/Validation/ValidatesRequests.php

```diff
@@ -23,11 +23,11 @@
      *
      * @param  \Illuminate\Contracts\Validation\Validator|array  $validator
      * @param  \Illuminate\Http\Request|null  $request
-     * @return void
+     * @return array
      */
     public function validateWith($validator, Request $request = null)
     {
-        $request = $request ?: app('request');
+        $request = $request ?: request();
```

Comment:
Why did this line need to change? I'm trying to understand the difference before/after?